xline/src/storage/db.rs

```diff
@@ -89,6 +91,15 @@ where
         })
     }
 
+    fn get_snapshot(&self) -> Result<Box<dyn SnapshotApi>, ExecuteError> {
+        let path = format!("/tmp/xline_snapshot_{}", uuid::Uuid::new_v4());
```

Comment:
Is this `path` temporary? Shall we make it configurable?

Reply:
yes, it is, after replying to the client, the server does not need to save these files